internal/server/server.go

```diff
@@ -126,14 +128,34 @@ func startServer(cfg *jailConfig, errCh chan<- error) {
 	}
 }
 
-func runServer(cfg *jailConfig) error {
+func execProxy(cfg *config.Config) error {
 	runtime.LockOSThread()
 	defer runtime.UnlockOSThread()
-	if err := dropPrivs(cfg); err != nil {
+	if err := privs.DropPrivs(cfg); err != nil {
 		return err
 	}
-	if err := unix.Exec("/jail/run", []string{"run", "server"}, os.Environ()); err != nil {
+	if err := unix.Exec("/jail/run", []string{"run", "proxy"}, os.Environ()); err != nil {
 		return fmt.Errorf("exec run: %w", err)
 	}
 	return nil
 }
+
+func RunProxy(cfg *config.Config) error {
+	errCh := make(chan error)
+	go runNsjailChild(errCh)
+	go startProxy(cfg, errCh)
+	return <-errCh
+}
+
+func ExecServer(cfg *config.Config) error {
+	if cfg.Pow > 0 {
+		if err := execProxy(cfg); err != nil {
+			return err
+		}
+	} else {
+		if err := execNsjail(cfg); err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
(nit) Given these were originally just in jailrun.go and this file is already fairly large, should we put these external entrypoints into this package into a different file? That way also the files are split into only exported and only unexported functions.